index.html

```diff
@@ -35,10 +43,14 @@ <h1 class="intro">I'm Max <br> Glad to see you</h1>
       <div class="social">
         <span class="fa fa-twitter twitter"></span>
         <span class="fa fa-linkedin"></span>
-        <span class="fa fa-medium-m"></span>
+        <span class="fa fa-medium"></span>
         <span class="fa fa-github"></span>
-        <span class="fa-forumbee"></span>
+        <span class="fa fa-hand-peace"></span>
```

Comment:
- [ ] [REQUIRED] It will be nice for you to ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)